### PR TITLE
Resource view

### PR DESCRIPTION
**Pull request by kdimopulu**

### CODE REVIEW

File: staff_features/resources/step_definitions/resource_shared.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    break if rows.length == 1
+  end
+end
+
+Then 'the Resource is in the search results' do
+  expect(page).to have_css('tr', text: @uuid)
+end
+
+Then 'the Resource view page is displayed' do
+  expect(find('h2').text).to eq "Resource #{@uuid} Resource"
```
Comment (brianzelip) [inline staff_features/resources/step_definitions/resource_shared.rb:29]: I think adding another constraint here is better than just a heading (I know the existing tests often look for only a heading), maybe expecting `'.record-toolbar button', text: Edit`, or expecting the exact URL, etc.

**Review by brianzelip — CHANGES_REQUESTED** [review]
@kdimopulu looks good. Again below is a comment about adding constraints to an expectation.

File: staff_features/resources/step_definitions/resource_shared.rb (hunk 1 above)
Comment (brianzelip) [reply staff_features/resources/step_definitions/resource_shared.rb:29]: Walking this suggestion back based on https://github.com/archivesspace/e2e-tests/pull/41#discussion_r1892359072

File: staff_features/resources/step_definitions/resource_view.rb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  expect(search_result_rows.length).to eq 2
+  expect(search_result_rows[1]).to have_text @resource_a_uuid
+  expect(search_result_rows[0]).to have_text @resource_b_uuid
+end
+
+Then 'the two Resources are displayed sorted by ascending identifier' do
+  search_result_rows = all('#tabledSearchResults tbody tr')
+
+  expect(search_result_rows.length).to eq 2
+  expect(search_result_rows[1]).to have_text @accession_a_uuid
```
Comment (balmas) [inline staff_features/resources/step_definitions/resource_view.rb:83]: should accession_a_uuid be before accession_b_uid if sorted by ascending identifier?

**Pull request merged**